crossbar/adapter/rest/subscriber.py

```diff
@@ -59,21 +59,29 @@ def onJoin(self, details):
         subscriptions = self.config.extra["subscriptions"]
 
         debug = self.config.extra.get("debug", False)
-        method = self.config.extra.get("method", u"POST").encode("utf8")
+        method = self.config.extra.get("method", u"POST")
         expectedCode = self.config.extra.get("expectedcode")
 
         @inlineCallbacks
         def on_event(url, *args, **kwargs):
 
             headers = Headers({
+                # FIXME: are keys/values supposed to be byte strings?
```

Comment:
Not sure I get it: so it should read like the following already _now_? For this PR?

```python
headers = Headers({
    u"Content-Type": [u"application/json"]
})
```